service/src/main/java/org/sunbird/service/user/impl/ExtendedUserProfileServiceImpl.java

```diff
@@ -22,8 +22,13 @@ public void validateProfile(Request userRequest) {
                 Map<String, Object> profileDetails = (Map<String, Object>) userRequest.get(JsonKey.PROFILE_DETAILS);
                 if (profileDetails.get(JsonKey.PERSONAL_DETAILS) != null) {
                     Map<String, Object> personalDetails = (Map<String, Object>) profileDetails.get(JsonKey.PERSONAL_DETAILS);
-                    if (personalDetails.get(JsonKey.FIRST_NAME) != null) {
-                        personalDetails.put(JsonKey.FIRST_NAME, formatFirstName((String) profileDetails.get(JsonKey.FIRST_NAME)));
+                    if (personalDetails.get(JsonKey.FIRST_NAME) != null || personalDetails.get(JsonKey.FIRST_NAME_LOWER_CASE) != null) {
```

Comment:
you need to check stringutils is not blank -- instead of simply not null condition... 

what will happen if first_name is empty ??

Reply:
if firstName is empty as of now i am not updating it, bcz the request body for update may or may not have normal firstName outside profileDetails.. so if we need to add firatname proper value inside profileDetails we may need to do additional logic to read users data..